internal/dnfjson/dnfjson_test.go

```diff
@@ -383,113 +383,113 @@ func TestMakeDepsolveRequest(t *testing.T) {
+		rpmmd.PackageSpec{Name: "tzdata", Epoch: 0x0, Version: "2021e", Release: "1.el9", Arch: "noarch", RemoteLocation: "%s/Packages/tzdata-2021e-1.el9.noarch.rpm", Checksum: "sha256:42d89577a0f887c4baa162250862dea2c1830b1ced56c45ced9645ad8e2a3671", Secrets: "", CheckGPG: false, IgnoreSSL: true, Reason: "dependency", ReasonGroup: ""},
+		rpmmd.PackageSpec{Name: "util-linux", Epoch: 0x0, Version: "2.37.2", Release: "1.el9", Arch: "x86_64", RemoteLocation: "%s/Packages/util-linux-2.37.2-1.el9.x86_64.rpm", Checksum: "sha256:4ca41a925461daa936db284a59bf325ea061cdb39d8738e288cc19afe30a8ae8", Secrets: "", CheckGPG: false, IgnoreSSL: true, Reason: "dependency", ReasonGroup: ""},
+		rpmmd.PackageSpec{Name: "util-linux-core", Epoch: 0x0, Version: "2.37.2", Release: "1.el9", Arch: "x86_64", RemoteLocation: "%s/Packages/util-linux-core-2.37.2-1.el9.x86_64.rpm", Checksum: "sha256:0313682867c1d07785a6d02ff87e1899f484bd1ce6348fa5c673eca78c0da2bd", Secrets: "", CheckGPG: false, IgnoreSSL: true, Reason: "dependency", ReasonGroup: ""},
+		rpmmd.PackageSpec{Name: "vim-minimal", Epoch: 0x2, Version: "8.2.2637", Release: "11.el9", Arch: "x86_64", RemoteLocation: "%s/Packages/vim-minimal-8.2.2637-11.el9.x86_64.rpm", Checksum: "sha256:ab6e48c8a118bed88dc734aaf21e743b57e94d448f9e38745c3b777af96809c7", Secrets: "", CheckGPG: false, IgnoreSSL: true, Reason: "user", ReasonGroup: ""},
+		rpmmd.PackageSpec{Name: "xz", Epoch: 0x0, Version: "5.2.5", Release: "7.el9", Arch: "x86_64", RemoteLocation: "%s/Packages/xz-5.2.5-7.el9.x86_64.rpm", Checksum: "sha256:b1c2d99961e50bb46400caa528aab9c7b361f5754427fd05ae22a7b551bf2ce5", Secrets: "", CheckGPG: false, IgnoreSSL: true, Reason: "dependency", ReasonGroup: ""},
+		rpmmd.PackageSpec{Name: "xz-libs", Epoch: 0x0, Version: "5.2.5", Release: "7.el9", Arch: "x86_64", RemoteLocation: "%s/Packages/xz-libs-5.2.5-7.el9.x86_64.rpm", Checksum: "sha256:770819da28cce56e2e2b141b0eee1694d7f3dcf78a5700e1469436461399f001", Secrets: "", CheckGPG: false, IgnoreSSL: true, Reason: "dependency", ReasonGroup: ""},
+		rpmmd.PackageSpec{Name: "zlib", Epoch: 0x0, Version: "1.2.11", Release: "31.el9", Arch: "x86_64", RemoteLocation: "%s/Packages/zlib-1.2.11-31.el9.x86_64.rpm", Checksum: "sha256:1c59b113fda8863e9066cc5d01c6d00bd9c50c4650e1c5b932082c8886e185d1", Secrets: "", CheckGPG: false, IgnoreSSL: true, Reason: "dependency", ReasonGroup: ""},
+		rpmmd.PackageSpec{Name: "zsh", Epoch: 0x0, Version: "5.8", Release: "7.el9", Arch: "x86_64", RemoteLocation: "%s/Packages/zsh-5.8-7.el9.x86_64.rpm", Checksum: "sha256:133da157fbd2b43e4a41af3ba7bb5267cf9ebed0aaf8124a76e5eca948c37572", Secrets: "", CheckGPG: false, IgnoreSSL: true, Reason: "user", ReasonGroup: ""},
```

Comment:
Why did the epoch change to hex for the test data?  Did you dump the result and copy it back in?
Not an issue, though.  Safe to ignore.

Reply:
I did. @thozza pointed it out as well and I've converted it back to decimal for the literal.